src/series.py

```diff
@@ -155,13 +155,18 @@ def _wait_for_seasons(self) -> None:
 
     def validate_show(self) -> None:
         """set show metadata"""
-        ta_channel: TAChannel = self._get_ta_channel()
+        ta_channel: TAChannel | None = self._get_ta_channel()
+        if ta_channel is None:
+            return
         self.update_metadata(ta_channel)
         self.update_artwork(ta_channel)
 
-    def _get_ta_channel(self) -> TAChannel:
+    def _get_ta_channel(self) -> TAChannel | None:
         """get ta channel metadata"""
-        episode: JFEpisode = self._get_all_episodes(limit=1)[0]
+        episodes: list[JFEpisode] = self._get_all_episodes(limit=1)
+        if(len(episodes) == 0):
```

Comment:
I don't think you need to do that here, if episodes is empty, this will already be false. You you should be able to do directly:
```python
if not episodes:
    return
```

`None` is the default return value from a function that doesn't explicitly return something, so you should be good with that.